pkgs/development/go-modules/generic/default.nix

```
@@ -55,11 +57,24 @@
 with builtins;
 
 assert goPackagePath != "" -> throw "`goPackagePath` is not needed with `buildGoModule`";
+assert (vendorSha256 == "_unset" && vendorHash == "_unset") -> throw "either `vendorSha256` or `vendorHash` is required";
```

Please put `vendorHash` first in the message.

```suggestion
assert (vendorSha256 == "_unset" && vendorHash == "_unset") -> throw "either `vendorSha256` or `vendorHash` is required";
```